src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx

```diff
@@ -12,36 +12,103 @@ redirects:
 </Callout>
 
 ## Set up incident intelligence [#get-started]
 
-To enable incident intelligence, follow these steps. Afterwards, issues should start to appear in your issue feed.
+To enable incident intelligence and start seeing issues in  your issue feed, follow these steps.
 
 * [1. Configure your environment (one-time)](#1-configure-environment).
 * [2. Configure sources](#1-configure-sources).
 * [3. Configure destinations](#2-configure-destinations).
 * [4. Configure workflows](#3-configure-workflows).
 
-## 1. Configure your environment (one-time) [#1-configure-environment]
+## 1. Configure your environment [#1-configure-environment]
+
+An environment groups together a set of one or more accounts. This allows you to see cross-account actions such as correlation, configuration, and data in one place. Everyone on your team needs access to your environment account and be authorized to view the data in it.
+
+By default, when you create an account, it's added to its own environment. 
+
+If you need cross-account collaboration, you can add multiple accounts into a single custom environment. You should only do this if you need to track cross-account data.
+
+By default, each account is in its own environment if it was created after November 2021. This is the recommended setting. 
+
+### What's stored in the environment? [#environment-detail]
+
+<img
+  title="The incident intelligence environment."
+  alt="A screenshot of the incident intelligence environment dropdown."
+  src={environment}
+/>
+
+<figcaption>
+  Use the environment dropdown to see your cross-account applied intelligence activity.
+</figcaption>
+
+* Your workflows configuration
+* Your decisions configuration
+* Your incidents
+* Issues created from your incidents
 
-To set up an environment in incident intelligence, you need an administrator to select a New Relic account for it. This account should be the one your team is using.
+Workflow enricher queries are executed at the environment account level.
 
-* **Who sets the environment?** Only administrators, and only for accounts where they have admin privileges.
-* **Can administrators set more than one environment?** They can set one environment per parent account and its child accounts. More than one can be set if an administrator has privileges for more than one parent account.
+### What's stored in the individual account? [#account-detail]
 
-<Callout variant="tip">
-  Incident intelligence is a cross-account product. This means you can send in data from any New Relic account or external source to correlate events.
+<img
+  title="The incident intelligence individual account."
+  alt="A screenshot of the incident intelligence individual account dropdown."
+  src={accountTeam}
+/>
+
+<figcaption>
+  Use the individual account dropdown to see policies, conditions, and destinations related to your account.
+</figcaption>
+
+* Your policies and conditions
```

Comment:
think we need some sort of intro sentence to introduce these bulletpoints. I assume they're meant to be answering the heading question of "what's store in..." but they're pretty far away from each other.

Reply:
Good point